clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactory.java

```diff
@@ -98,6 +108,7 @@ public void create(final String identifier, final ServerSideConfiguration config
             try {
               EhcacheClientEntity entity = ref.fetchEntity();
               try {
+                entity.setTimeouts(entityTimeouts);
```

Comment:
Could be passed as a parameter to `configure` (and `validate`) ... is that what you'd prefer?